benches/bench_for_estimates.rs

```diff
@@ -0,0 +1,287 @@
+#![allow(bare_trait_objects)]
+#![allow(non_snake_case)]
+#![feature(test)]
+extern crate rustfft;
+extern crate test;
+
+use paste::paste;
+use rustfft::algorithm::*;
+use rustfft::num_complex::Complex;
+use rustfft::num_traits::Zero;
+use rustfft::{Fft, FftDirection};
+use std::sync::Arc;
+use test::Bencher;
+
+// Make fft using planner
+fn bench_planned(b: &mut Bencher, len: usize) {
+    let mut planner = rustfft::FftPlannerScalar::new();
+    let fft: Arc<dyn Fft<f64>> = planner.plan_fft_forward(len);
+
+    let mut buffer: Vec<Complex<f64>> = vec![Complex::zero(); len];
+    let mut scratch: Vec<Complex<f64>> = vec![Complex::zero(); fft.get_inplace_scratch_len()];
+    b.iter(|| {
+        fft.process_with_scratch(&mut buffer, &mut scratch);
+    });
+}
+
+// Make fft using planner
+fn bench_planned_multi(b: &mut Bencher, len: usize, reps: usize) {
+    let mut planner = rustfft::FftPlannerScalar::new();
+    let fft: Arc<dyn Fft<f64>> = planner.plan_fft_forward(len);
+    let mut buffer: Vec<Complex<f64>> = vec![Complex::zero(); len * reps];
+    let mut output = buffer.clone();
+    let mut scratch: Vec<Complex<f64>> = vec![Complex::zero(); fft.get_outofplace_scratch_len()];
+    b.iter(|| {
+        fft.process_outofplace_with_scratch(&mut buffer, &mut output, &mut scratch);
+    });
+}
+
+// Make a radix4
+fn bench_radix4(b: &mut Bencher, len: usize) {
+    assert!(len % 4 == 0);
+
+    let fft = Radix4::new(len, FftDirection::Forward);
+
+    let mut buffer = vec![
+        Complex {
```

Comment:
Is there any reason not to use Complex::zero() here? It would cut out some extra lines and keep it consistent with the rest of the benchmarks.

Reply:
This is just a leftover from some copy-paste, will change it!